### ISSUE TITLE
Quarkus versions and quarkus logging versions

### ISSUE THREAD

**Issue by wanderingfella** [quarkiverse/quarkus-logging-json#112]
Apologies, if it's a basic question, I am very new to Quarkus, and immediately needed this feature, so it's one of the first things I am looking at.

I seem to be having issues getting specific versions of Quarkus working with the right version of quarkiverse logging json.
I have tried a number of permutations, but seem to get different issues depending on the combination/exclusions.

The closest I have is

    implementation 'io.quarkiverse.loggingjson:quarkiverse-logging-json:0.2.3'

with Qaurkus version 1.13.7.Final.

I haven't included io.quarkus:quarkus-logging-json currently, as they seem to clash.

In short could a sample dependency set be provided that work together?

Thanks!

**Comment by dfa1** [comment]
hello :)

I'm using version 1.0.1 with Quarkus 1.13.7 without issues (but I'm using maven). 
Beware: versions before 1.0.1 have a nasty race condition fixed recently by https://github.com/quarkiverse/quarkus-logging-json/commit/684bf6458f0791a5c8c1f46aa70329a348749513

**Comment by wanderingfella** [comment]
Thanks.

Things I have tried, all with version 1.13.7.Final of Qarkus

    implementation 'io.quarkus:quarkus-jackson'
  
Outcome : JSON logging works

Adding the below dependency, to quarkus logging

    implementation 'io.quarkiverse.loggingjson:quarkiverse-logging-json:0.2.3'

Outcome, doesn't work =>
java.lang.RuntimeException: java.lang.IllegalArgumentException: Multiple matching properties for name "log.console.json" property was matched by both boolean io.quarkiverse.loggingjson.Config.enable and boolean io.quarkus.logging.json.runtime.JsonConfig.enable. This is likely because you have an incompatible combination of extensions that both define the same properties (e.g. including both reactive and blocking database extensions)

Then removing 
  implementation 'io.quarkus:quarkus-logging-json'

Works, as in the logging comes out in JSON.

Next step, try to use key values...
 i.e.      LOG.info("message ", KeyValueStructuredArgument.kv("test", "2"));

Doesn't compile using,  
       import org.jboss.logging.Logger;

Switched to using sl4j

and it works!

Apologies.
